externs/olx.js

```diff
@@ -301,6 +301,49 @@ olx.MapOptions.prototype.target;
  */
 olx.MapOptions.prototype.view;
 
+/**
+ * The filter function will receive one argument, the
+ * {@link ol.layer.Layer layer-candidate} and it should return a boolean
+ * value.
+ * @typedef {(function(ol.layer.Layer): boolean)}
+ */
+olx.LayerFilterFunction;
```

Comment:
I introduced this typedef because i was not able to annotate it properly otherwise. I will see if i can remove this.